.changeset/cold-boats-share.md

```diff
@@ -0,0 +1,7 @@
+---
+"@lynx-js/chunk-loading-webpack-plugin": patch
```

Comment:
Please remember to remove this when reverting the change in chunk-loading-webpack-plugin.